### ISSUE TITLE
Error during Bowtie2 alignment process

### ISSUE THREAD

**Issue by chuafischer** [fredeBio/REALPHY#6]
Hi !

I don't know if this tool is still maintained so I try my luck.

Background: Testing Realphy by running it with a set of 11 samples of P. aeruginosa. I want to merge the alignment results for 3 different reference samples included in the set.

Problem: This error is occuring during bowtie2 mapping step (see below)

Questions: Any idea on how to deal with this issue ? As the file "ref_50fasta.fastq.gz" is not a FASTQ that is part of my data, but is a file created by RealPhy, I don't know what I could do on my side to solve issue. 
Also, why is this file produced ? Is it useful for the rest of the analyses ?

Thanks for your help !

Stdout:
```
/home/jpr/miniconda3/envs/realphy/bin
reference/ref1.fasta out/ref1/core/ref1.fas
Cut fasta sequence files into fragments...
reference/cut/ref1_50fasta.fastq.gz already exists. Continue with next file.
reference/cut/ref2_50fasta.fastq.gz already exists. Continue with next file.
reference/cut/ref3_50fasta.fastq.gz already exists. Continue with next file.
Running alignment program to align sequence fragments to reference...
/home/jpr/miniconda3/envs/realphy/bin/bowtie2 -x out/ref1/core/ref1.fas -U reference/cut/ref3_50fasta.fastq.gz  -S out/ref1/alignOut_NoGenes/ref3_50fasta.bam.temp -N 1  --no-unal  -a  -L 22  -p 48
Saw ASCII character 10 but expected 33-based Phred qual.
terminate called after throwing an instance of 'int'
Aborted
(ERR): bowtie2-align exited with value 134
Bowtie was not successful!
```

**Comment by fredeBio** [comment]
Hey Aurelie,
based on the error message my guess would be that your ref3 sequence, contains non-DNA values, or is not formatted right.

Cheers,
Frederic

----- Original Message -----
From: "Aurelie Fischer" ***@***.***>
To: "fredeBio/REALPHY" ***@***.***>
Cc: "Subscribed" ***@***.***>
Sent: Friday, September 22, 2023 1:24:00 PM
Subject: [fredeBio/REALPHY] Error during Bowtie2 alignment process (Issue #6)

Hi !

I don't know if this tool is still maintained so I try my luck.

Background: Testing Realphy by running it with a set of 11 samples of P. aeruginosa. I want to merge the alignment results for 3 different reference samples included in the set.

Problem: This error is occuring during bowtie2 mapping step (see below)

Questions: Any idea on how to deal with this issue ? As the file "ref_50fasta.fastq.gz" is not a FASTQ that is part of my data, but is a file created by RealPhy, I don't know what I could do on my side to solve issue. 
Also, why is this file produced ? Is it useful for the rest of the analyses ?

Thanks for your help !

Stdout:
```
/home/jpr/miniconda3/envs/realphy/bin
reference/ref1.fasta out/ref1/core/ref1.fas
Cut fasta sequence files into fragments...
reference/cut/ref1_50fasta.fastq.gz already exists. Continue with next file.
reference/cut/ref2_50fasta.fastq.gz already exists. Continue with next file.
reference/cut/ref3_50fasta.fastq.gz already exists. Continue with next file.
Running alignment program to align sequence fragments to reference...
/home/jpr/miniconda3/envs/realphy/bin/bowtie2 -x out/ref1/core/ref1.fas -U reference/cut/ref3_50fasta.fastq.gz  -S out/ref1/alignOut_NoGenes/ref3_50fasta.bam.temp -N 1  --no-unal  -a  -L 22  -p 48
Saw ASCII character 10 but expected 33-based Phred qual.
terminate called after throwing an instance of 'int'
Aborted
(ERR): bowtie2-align exited with value 134
Bowtie was not successful!
```

-- 
Reply to this email directly or view it on GitHub:
https://github.com/fredeBio/REALPHY/issues/6

**Comment by chuafischer** [comment]
Hi Frederic,

Thanks for your quick answer.
My ref3 is a multi-fasta file, the FASTA contains several contig sequences.

Could it be related to the problem ?

-Aurelie

**Comment by fredeBio** [comment]
No, this should be fine. But it probably contains non-DNA characters or something else weird.

----- Original Message -----
